editor/docks/groups_editor.h

```diff
@@ -122,6 +124,11 @@ class GroupsEditor : public VBoxContainer {
 
 	void _node_removed(Node *p_node);
 
+	void _add_to_group(const StringName &name, bool persist, const Array &nodes);
```

Comment:
```suggestion
	void _add_to_group(const StringName &p_name, bool p_persist, const Array &p_nodes);
```
Method arguments should be prefixed with `p_`.